Recast/Include/Recast.h

```diff
@@ -441,6 +441,29 @@ struct rcPolyMeshDetail
 	int ntris;				///< The number of triangles in #tris.
 };
 
+static const unsigned char RC_AREA_FLAGS_MASK = 0xFF;
```

Comment:
Ok, if you want to introduce this it should have value 0x3F. Sorry for the confusion.